front/app/containers/Admin/projects/project/permissions/granular_permissions/containers/Granular/ActionForm.tsx

```diff
@@ -128,7 +128,7 @@ const ActionForm = ({
                 selected={permittedBy === 'everyone'}
               />
             )}
-            {emailConfirmPermissionEnabled && (
```

Comment:
This option should always be shown to anyone who can see this form. It's already flagged with granular_permissions (phase-access rights). So this flag is not needed.